.github/workflows/build-container-images.yaml

```diff
@@ -71,3 +72,14 @@ jobs:
           tags: ghcr.io/rackerlabs/understack/argo-ironic-client-python3.11.8:latest,ghcr.io/rackerlabs/understack/argo-ironic-client-python3.11.8:${{ env.VERSION_PYTHON_IRONIC }}
           labels: |
               org.opencontainers.image.version=${{ env.VERSION_PYTHON_IRONIC }}
+
+      - name: Build and deploy Python 3.11 with Nautobot Int Sync
+        uses: docker/build-push-action@v5
+        with:
+          context: argo-workflows/nautobot-interface-sync
+          file: argo-workflows/nautobot-interface-sync/containers/Dockerfile.nautobot_int_sync
+          # push for all main branch commits
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ghcr.io/rackerlabs/understack/nautobot-interfaces-sync-python3.11.8:latest,ghcr.io/rackerlabs/understack/nautobot-interfaces-sync-python3.11.8:${{ env.VERSION_PYTHON_NAUTOBOT_INT_SYNC }}
```

Comment:
Just seems like it's gonna be really painful to have the Python version in the name. Why don't we just name the container for what it does? We aren't gonna have different Python versions of it.

Curious what you think @skrobul

Reply:
adjusted